src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/fee/FeeResponse.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.entities.fee;
+
+import static java.util.Objects.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
I think this should be a strategy for all our interactions with other services.  So this could be done centrally in the Spring JacksonConfiguration.  Can disable the feature `FAIL_ON_IGNORED_PROPERTIES` I think